src/main/java/sg/ncl/MainController.java

```diff
@@ -821,12 +836,9 @@ private void registerUserToDeter(JSONObject mainObject) throws
                 ExceptionState exceptionState = ExceptionState.parseExceptionState(error.getError());
 
                 switch (exceptionState) {
-                    case JOIN_PROJECT_EXCEPTION:
-                        log.warn("Register new users join team request : team name error");
-                        throw new TeamNotFoundException("Team name does not exists");
-                    case APPLY_NEW_PROJECT_EXCEPTION:
-                        log.warn("Register new users new team request : team name error");
-                        throw new ApplyNewProjectException();
+                    case ADAPTER_DETERLAB_OPERATION_FAILED_EXCEPTION:
+                        log.warn("Register new user failed on DeterLab: {}", error.getMessage());
+                        throw new DeterLabOperationFailedException("Error: " + (error.getMessage().contains("unknown error")? ERR_SERVER_OVERLOAD : error.getMessage()));
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Define a constant instead of duplicating this literal "Error: " 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1192)